app/src/pages/outside/loginPage/pageBlocks/serviceVersionsBlock/serviceVersionBlockTooltip/serviceVersionsBlockWithTooltip/serviceVersionsBlockWithTooltip.jsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2023 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import classNames from 'classnames/bind';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import styles from './serviceVersionsBlockWithTooltip.scss';
+import { ServiceVersionItem } from '../../serviceVersionItem';
+
+const cx = classNames.bind(styles);
+
+export const ServiceVersionsBlockWithTooltip = ({ services, className }) => {
```

Comment:
I suggest to rename this component to `ServiceVersionsBlockTooltip` or `ServiceVersionsTooltip` as this component is a tooltip component itself.
Please don't forget to rename all corresponding stuff.